src/platform/user/profile/utilities/index.js

```diff
@@ -122,12 +122,28 @@ export function mapRawUserDataToState(json) {
 // as a trigger to properly update any components that subscribe to it.
 export const hasSession = () => localStorage.getItem('hasSession');
 
+function compareLoginPolicy(loginPolicy) {
+  let attemptedLoginPolicy = sessionStorage.getItem(
+    authnSettings.PENDING_LOGIN_TYPE,
+  );
+
+  attemptedLoginPolicy =
+    attemptedLoginPolicy === 'mhv' ? 'myhealthevet' : attemptedLoginPolicy;
```

Comment:
Is there a reason we need to expand `mhv` if we'd normally report events like `login-success-mhv`?

Reply:
The backend reports the `serviceName` as `myhealthevet`, so I just expanded it to make the comparison.  GA has been using `login-success-mhv` so I didn't want to change the event for consistency.